packages/react-ui-map/src/Mapbox/Mapbox.js

```diff
@@ -24,30 +26,36 @@ export default class Mapbox extends PureComponent {
 
   static defaultProps = {
     theme: {},
+    flyOnCenterChange: false,
   }
 
   constructor(props, context) {
     super(props, context)
+    mapboxgl.accessToken = props.token
 
     this.state = {
       map: null,
     }
   }
 
   getChildContext() {
-    return { map: this.state.map }
+    return {
+      map: this.state.map,
+    }
   }
 
   componentDidMount() {
-    mapboxgl.accessToken = this.props.token
-    const map = new mapboxgl.Map({
+    const opts = {
       container: this.container,
       style: this.props.style,
       center: this.props.center,
       zoom: this.props.zoom,
       theme: this.props.theme,
-    })
+    }
 
+    if (!opts.center) delete opts.center
```

Comment:
This feels odd.  How about adding `center` to `opts` if it has a value.

Reply:
good idea